server/src/main/java/com/google/musicanalysis/site/YouTubeCallbackServlet.java

```diff
@@ -17,7 +24,12 @@ protected String getClientId() {
 
   @Override
   protected String getClientSecret() {
-    return System.getenv().get("YOUTUBE_CLIENT_SECRET");
+    try {
+      return Secrets.getSecretString("OUTUBE_CLIENT_SECRET");
```

Comment:
bug -- please change to "YOUTUBE_CLIENT_SECRET" instead of "OUTUBE_CLIENT_SECRET"